src/Prefix.jl

```diff
@@ -297,7 +344,11 @@ function install(tarball_url::AbstractString,
     if verbose
         Compat.@info("Installing $(tarball_path) into $(prefix.path)")
     end
-    
+
+    # remove old files if clean=true
+    manifest_path = manifest_from_url(tarball_url, prefix=prefix)
+    clean && isfile(manifest_path) && uninstall(manifest_path, verbose=verbose)
```

Comment:
I think we can roll the functionality of `clean` into `force`.  I think it is reasonable to expect that two different tarballs with the same base filename (and therefore the same manifest filename) will have similar contents; and thus without `force=true`, they will fail the check down below where we check to see if we're going to overwrite something.  Since this will almost always be the case if the manifests match (unless we have a freak occasion where and old `foo.tar.gz` and a new `foo.tar.gz` share no files in common), I think we should just check for `force` instead of `clean`.

Reply:
Okay, will do.